multiqc/modules/bcftools/stats.py

```diff
@@ -112,45 +128,56 @@ def parse_bcftools_stats(self):
                     'anchor': 'bcftools-stats',
                     'content': linegraph.plot(self.bcftools_stats_indels, pconfig)
                 })
+            # Make line graph of variants per depth
+            if len(depth_data) > 0:
+                import pprint
+                pprint.pprint(depth_data)
+                pconfig = {'id': 'bcftools_stats_depth', 'title': 'Variant depths',
+                           'ylab': 'Fraction of sites (%)', 'xlab': 'Variant depth',
+                           'ymin': 0, 'ymax': 100, 'categories': True}
+                self.sections.append({'name': 'Variant depths',
+                                      'anchor': 'bcftools-stats',
```

Comment:
This anchor needs to be unique and different to the plot above, otherwise the navigation links on the left of the report don't work.

I should probably implement a check for this in the code..